### PR TITLE
first issue: hard coded value into codehintlist

### PR DESCRIPTION
**Pull request by inesteixeira**
This is about issue #13637 
Working on this with FilipePintoReis and this is a temporary fix for what he described (for better understanding of the problem).

### CODE REVIEW

**Comment by xiaoyu0519** [comment]
good

File: src/editor/CodeHintList.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ define(function (require, exports, module) {
          *
          * @type {number}
          */
-        this.maxResults = ValidationUtils.isIntegerInRange(maxResults, 1, 1000) ? maxResults : 50;
+        this.maxResults = 500;
```
Comment (navch) [inline src/editor/CodeHintList.js]: This is already there you can change maxhints from preference. (i.e. "maxCodeHints": 100)

We are taking this value from preference only, Please check CodeHintManager.js
"maxCodeHints = PreferencesManager.get("maxCodeHints");"

**Comment by navch** [comment]
@inesteixeira If my comment solves your problem then please close this PR.

**Comment by navch** [comment]
As we already have a preference for maxCodeHint so closing this PR.

**Pull request closed without merging**